bblfsh/client.py

```diff
@@ -99,6 +100,14 @@ def native_parse(self, filename, language=None, contents=None, timeout=None):
                                language=self._scramble_language(language))
         return self._stub.NativeParse(request, timeout=timeout)
 
+    def supported_languages(self):
+        sup_response = self._stub.SupportedLanguages(SupportedLanguagesRequest())
+
+        try:
+            return sup_response.ListFields()[1][1]
```

Comment:
Maybe a dumb question, but why it's an index access instead of field access?

Reply:
Brainfart :) I was testing it the response could be accessed with that method as a normal python indexed list and the objects and dictionaries and forgot to restore the access to languages.

I'll change it in the next commit + a unittest.